### PR TITLE
Updated Fabric Installer to 0.11.2

### PR DESCRIPTION
**Pull request by ghost**
- Updated to Fabric Installer 0.11.2
- Removed "Standalone Server Installer" notice on Server tab, as it links back to FabricMC.
- Updated name to `Legacy Fabric Installer`
- Updated "Fabric API" language asset to "Legacy Fabric API"

### CODE REVIEW

File: src/main/java/net/fabricmc/installer/Main.java [hunk 1]
```diff
@@ -77,7 +77,7 @@ public static void main(String[] args) throws IOException {
 			HANDLERS.forEach(handler -> System.out.printf("%s %s\n", handler.name().toLowerCase(), handler.cliHelp()));
 			loadMetadata();
 
-			System.out.printf("\nLatest Loader: %s\n", Main.LOADER_META.getLatestVersion(false).getVersion());
+			System.out.printf("\nLatest Version: %s\nLatest Loader: %s\n", GAME_VERSION_META.getLatestVersion(argumentParser.has("snapshot")).getVersion(), Main.LOADER_META.getLatestVersion(false).getVersion());
```
Comment (thecatcore) [inline src/main/java/net/fabricmc/installer/Main.java]: Latest Version is not applicable to Legacy Fabric.

File: src/main/java/net/fabricmc/installer/Main.java [hunk 2]
```diff
@@ -41,7 +41,7 @@ public static void main(String[] args) throws IOException {
 			System.setProperty("javax.net.ssl.trustStoreType", "WINDOWS-ROOT");
 		}
 
-		System.out.println("Loading LegacyFabric Installer: " + Main.class.getPackage().getImplementationVersion());
+		System.out.println("Loading Fabric Installer: " + Main.class.getPackage().getImplementationVersion());
```
Comment (thecatcore) [inline src/main/java/net/fabricmc/installer/Main.java]: Why change everything else to LegacyFabric and revert back this one?

File: src/main/java/net/fabricmc/installer/server/ServerInstaller.java [hunk 3]
```diff
@@ -88,8 +88,8 @@ public static void install(Path dir, LoaderVersion loaderVersion, String gameVer
 
 			mainClassMeta = json.at("mainClass").asString();
 		} else { // loader jar available, generate library list from it
-			libraries.add(new Library(String.format("net.fabricmc:fabric-loader:%s", loaderVersion.name), "https://maven.fabricmc.net/", loaderVersion.path));
-			libraries.add(new Library(String.format("net.fabricmc:intermediary:%s", gameVersion), "https://maven.legacyfabric.net/", null));
+			libraries.add(new Library(String.format("net.fabricmc:fabric-loader:%s", loaderVersion.name), null, loaderVersion.path));
+			libraries.add(new Library(String.format("net.fabricmc:intermediary:%s", gameVersion), "https://maven.fabricmc.net/", null));
```
Comment (thecatcore) [inline src/main/java/net/fabricmc/installer/server/ServerInstaller.java]: Why again? Legacy Fabric intermediaries are on our maven, not on official fabric one

**Review by thecatcore — CHANGES_REQUESTED** [review]

**Comment by mclegoman** [comment]
I apologize for the overlooked changes, they should be fixed now.
In hindsight, I could have compared the changes with the upstream version directly on GitHub but oh well.

**Review by thecatcore — APPROVED** [review]

**Pull request merged**